solid-cpp/Solid_solver.cpp

```diff
@@ -0,0 +1,136 @@
+#include "precice/SolverInterface.hpp"
+#include <iostream>
+#include <stdlib.h>
+#include "mpi.h"
+#include "SolidSolver.h"
```

Comment:
I see a naming inconsistency here:
- `SolidSolver.h`
- `Solid_solver.cpp`
- `SolidComputeSolution.cpp`
- `fluid_solver.cpp`

I guess that they should be:
- `fluid_solver.cpp` and `solid_solver.cpp`, or `FluidSolver.cpp` and `SolidSolver.cpp`
- the respective headers should be `fluid_solver.h` and `solid_solver.h`, or `FluidSolver.h` and `SolidSolver.h`
- not sure if we really need the `...ComputeSolution.cpp` files to be separate.

Of course, we should not rewrite all the internals here, but I think the file names are important for this PR.

Reply:
Good point! I decided to go for the capital option (aka `FluidSolver.cpp`)